sbi/analysis/sensitivity_analysis.py

```diff
@@ -501,3 +502,53 @@ def project(self, theta: Tensor, num_dimensions: int) -> Tensor:
         projected_theta = torch.mm(theta, projection_mat)
 
         return projected_theta
+
+
+def posterior_shrinkage(prior_std, post_std):
+    """
+    Calculate the posterior shrinkage. The posterior shrinkage
+    quantifies how much the posterior distribution contracts
+    from the initial prior distribution.
+
+    Parameters
+    ----------
+    prior_std : float, array-like
+        The standard deviation of the prior distribution.
+    post_std : float, array-like
+        The standard deviation of the posterior distribution.
+
+    returns
+    -------
+    shrinkage : float, array-like
+        The posterior shrinkage.
+
+    """
+    return 1 - (post_std / prior_std) ** 2
+
+
+def posterior_zscore(true_mean, post_mean, post_std):
```

Comment:
Same as above i.e. type hints here.

And the metric should be suitable for vector input.